inference-engine/src/transformations/src/transformations/common_optimizations/common_optimizations.cpp

```diff
@@ -116,6 +117,7 @@ bool ngraph::pass::CommonOptimizations::run_on_function(std::shared_ptr<ngraph::
     decomp->add_matcher<ngraph::pass::ConvertSpaceToDepth>();
     decomp->add_matcher<ngraph::pass::BatchNormDecomposition>();
     decomp->add_matcher<ngraph::pass::BatchNormV5Decomposition>();
+    decomp->add_matcher<ngraph::pass::MVN6Decomposition>();
```

Comment:
why do we want to decompose MVN6 here? I think it is a matter of plugin specific.
Also, if we do have a reason to decompose it, so why don't we decompose the previous versions of MVN then?

Reply:
We decided to enable decomposition for the cases when plugins do not support MVN-6 yet, it is an unlikely scenario since all plugins committed to supporting MVN-6, but this is done to not break anything that worked before during the transition.

Reply:
Then I propose to add a comment that now it is triggered temporarily while transition period. Because it is definitely a plugin matter if it doesn't support it must care of decomposition on its own.

Reply:
I expect that the "transition time" may be very long :). Also, following the same proposal we should add a comment for each and every decomposition transformation above. So let's not do this.